src/handlers/Chrome67.ts

```diff
@@ -541,10 +541,20 @@ export class Chrome67 extends HandlerInterface
 
 		const parameters = rtpSender.getParameters();
 
-		parameters.encodings.forEach((encoding: RTCRtpEncodingParameters, idx: number) =>
+		if (Array.isArray(params)) 
 		{
-			parameters.encodings[idx] = { ...encoding, ...params };
-		});
+			parameters.encodings.forEach((encoding: RTCRtpEncodingParameters, idx: number) =>
+			{
```

Comment:
This is wrong. Old matching properties override the new ones.

Reply:
This is the way property assignment was done before. It seems to be correct:
```js
const oldObj = { x: 1 }
const newObj = { x: 2 }
{ ...oldObj, ...newObj } // { x: 2 }
```

Reply:
I'm in mobile and could be wrong. I'll review in next days. Thanks